galasa-parent/dev.galasa.framework.api.authentication/src/main/java/dev/galasa/framework/api/authentication/internal/routes/AuthTokensRoute.java

```diff
@@ -58,8 +62,14 @@ public class AuthTokensRoute extends BaseRoute {
     // Regex to match /auth/tokens and /auth/tokens/ only
     private static final String PATH_PATTERN = "\\/tokens\\/?";
 
+    private static final String REST_API_CLIENT = "rest-api";
+    private static final String WEB_UI_CLIENT = "web-ui";
+
     private static final IBeanValidator<TokenPayload> validator = new TokenPayloadValidator();
 
+    private ITimeService timeService ;
+
+    // TODO: Need to be passed a framework so we can get a timer service from it.
```

Comment:
Does this still need doing? I don't think the framework needs to be passed in, just an ITimeService

Reply:
The unit tests don't seem to be using a mock time service since this constructor sets a `SystemTimeService()`, so we might run into time-related assertion errors